R/build-articles.R

```diff
@@ -55,7 +55,7 @@ build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L, encodi
   on.exit(set_pkgdown_env(old))
 
   pkg <- as_pkgdown(pkg)
-  path <- rel_path(path, pkg$path)
+  path <- rel_path(path)
```

Comment:
Agree with it. I have got wrong, parameter `path`  is just the path relative to package path. Keep the pkg website in  the subdirectory `docs`   of the pkg root directory is the most common and best practice. 

However, it will raise an error while the package path is not `.` (e.g.  `pkgname`), which was caused by func `rel_path`:

```r
# pkg path  "packagename"
rel_path('pkgname/docs','pkgname')
## pkgname/pkgname/docs 
```